packages/itwinui-css/src/code/codeblock.scss

```diff
@@ -39,21 +38,21 @@
       white-space: nowrap;
       hyphens: auto;
       counter-increment: section;
-      font-family: $iui-monospace;
+      font-family: var(--iui-font-mono);
       background-color: var(--iui-color-background-1);
 
       &:nth-child(even) {
-        background: rgba(var(--iui-color-foreground-body-rgb), 0.02);
+        background: var(--iui-color-background-1-hover);
```

Comment:
Why on dark theme these got inverted? 
![image](https://user-images.githubusercontent.com/36186912/185617581-d059e9df-c012-48ee-a373-9ed0e3fc9376.png)

Reply:
Should be fixed now